reports/templegold-report.md

```diff
@@ -281,15 +281,93 @@ Consider protecting the initializer functions with modifiers.
 
 Downcasting int/uints in Solidity can be unsafe due to the potential for data loss and unintended behavior.When downcasting a larger integer type to a smaller one (e.g., uint256 to uint128), the value may exceed the range of the target type,leading to truncation and loss of significant digits. Use OpenZeppelin's SafeCast library to safely downcast integers.
 
-<details><summary>2 Found Instances</summary>
+<details><summary>15 Found Instances</summary>
+
+
+- Found in contracts/util/ABDKMath64x64.sol [Line: 93](../tests/2024-07-templegold/protocol/contracts/util/ABDKMath64x64.sol#L93)
+
+	```solidity
+	      return int128 (result);
+	```
+
+- Found in contracts/util/ABDKMath64x64.sol [Line: 121](../tests/2024-07-templegold/protocol/contracts/util/ABDKMath64x64.sol#L121)
+
+	```solidity
+	      return int128 (result);
+	```
+
+- Found in contracts/util/ABDKMath64x64.sol [Line: 136](../tests/2024-07-templegold/protocol/contracts/util/ABDKMath64x64.sol#L136)
+
+	```solidity
```

Comment:
These are inside `unchecked` blocks . Weren't caught before, they are now !

Reply:
These are actually false positives, but we should tackle them in a separate issue/pr